docker-compose.yaml

```diff
@@ -3,14 +3,7 @@ version: '2.1'
 services:
 
   base:
-    image: upm-base
-    build:
-      context: .
-      dockerfile: docker/Dockerfile.base
-      args:
-        - http_proxy
-        - https_proxy
-        - no_proxy
+    image: dnoliver/upm-base:upm-travis-docker-pull
```

Comment:
This is temporary until https://github.com/dnoliver/iot-devkit-docker/pull/2 gets merged into master. I didn't want to brake another build by publishing a master image that will only be used by this branch. So travis automatically publish the docker image with the branch name as tag, and I can use it from here.

I will update the image to "latest" before merging to master